### ISSUE TITLE
Meeting details to appear between Sources of Information and actual references

### ISSUE THREAD

**Issue by ricardd** [pbs-assess/csasdowntex#104]
I am told that the information about the Science Response meeting should appear between Sources of Information and the actual references.

So in the case of the draft SR from csasdown it means that the sentence "Optional for regular Science Responses, and mandatory for Stock Status Updates as these must include the citation for the peer-reviewed Science Advisory Report." should appear after Sources of information and before the Francis (2011) citation.

**Comment by ricardd** [comment]
The pagination is fine with a Word output.

**Comment by ricardd** [comment]
@grinnellm Sean tells me that you are working on a Science Response too, could you please see if there is a fix for this issue? I assume that you will be facing the same requirement from CSAS.

**Comment by grinnellm** [comment]
Hey @ricardd, good to hear from you. Yes we just published our herring assessment SR ([English](http://www.dfo-mpo.gc.ca/csas-sccs/Publications/ScR-RS/2020/2020_004-eng.html) and [French](http://www.dfo-mpo.gc.ca/csas-sccs/Publications/ScR-RS/2020/2020_004-fra.html) versions) using **csasdown** and it went quite smoothly. It seems like it was thoroughly reviewed by Ann here in the Pacific, and by Laura in the NCR. We did not include the meeting details between Sources of information and the actual references; I wasn't aware of this requirement and CSAS did not bring it up for this report, or for our SR last year. The only place that we mention the meeting is in the last sentence/paragraph of the Context, which reads:
> This Science Response results from the Science Response Process of September 30, 2019 on the Stock status update with application of management procedures for BC Pacific Herring: 2019 status and 2020 forecast.
